### PR TITLE
added option to ignore params that are not allowed in create/update

### PR DESCRIPTION
**Pull request by gniquil**
This is for issue #368

### CODE REVIEW

**Comment by barelyknown** [comment]
Code looks good. Can you fix the merge conflicts with master and push the update?

File: lib/jsonapi/acts_as_resource_controller.rb [hunk 1]
```diff
@@ -121,6 +122,14 @@ def base_response_meta
       {}
     end
 
+    def base_meta
+      if @request.nil?
+        base_response_meta
+      else
+        @request.warnings.empty? ? base_response_meta : base_response_meta.merge(:warnings => @request.warnings)
```
Comment (lgebhardt) [inline lib/jsonapi/acts_as_resource_controller.rb]: Could you please change to the newer ruby hash syntax: `warnings: @request.warnings`

**Comment by barelyknown** [comment]
Nice PR. Thanks. :+1:

**Comment by lgebhardt** [comment]
Thanks @gniquil and @barelyknown. FYI I added the option to the configuration section of the Readme.

**Comment by barelyknown** [comment]
Ah, I should have caught that. Thanks.

**Comment by lgebhardt** [comment]
No problem. I missed it earlier too.

**Comment by gniquil** [comment]
:+1:

**Pull request merged**